.github/workflows/api_diff_report.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Setup python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.7
+          python-version: 3.10
```

Comment:
weird that it parses as 3.1? Maybe it needs to be "3.10"?

Reply:
I think you're right: https://github.com/actions/setup-python#basic-usage

When I first read the error message I thought I typo-ed.